kitsune/sumo/static/sumo/scss/form-wizard.scss

```diff
@@ -0,0 +1,45 @@
+:root,
+:host {
+  --color-progress: var(--color-blue-06);
+}
+
+:host {
+  display: flex;
+  flex-direction: column;
+  border-radius: 8px;
+  box-shadow: 0px 2px 6px rgba(58, 57, 68, 0.2);
+}
+
+.form-wizard-content {
+  padding-inline: 32px;
+  padding-block-start: 24px;
+}
+
+h2 {
+  margin: 0;
+  font-size: 1.5rem;
+  line-height: 1;
+  color: var(--color-heading);
+}
+
+ul {
+  margin-inline-end: 48px;
+}
+
+section {
+  display: flex;
+}
+
+progress {
```

Comment:
We should reach out to Tati and make sure this wasn't meant to animate. The progress bar in about:welcome, for example, is constructed using two <div>'s and some clever CSS transition properties. I don't think we need to worry about that for this version, but it might be worth asking about as follow-up work.

Reply:
Good call! Left a comment on the Figma mocks to start the discussion there: https://www.figma.com/file/qe9yX8x51jjbt6DJYgayg2?node-id=3362:38070#444445016